### PR TITLE
9-view reviewsByCategory, uses reactContext

### PR DESCRIPTION
**Pull request by amnioc**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -24,6 +25,34 @@ function App() {
           path="/reviews/:review_id"
           element={<SingleReview reviews={reviews} />}
         />
+        <Route
```
Comment (itskatherine) [inline src/App.js:28]: This works well, but if you wanted to add another category you would have to manually add another route to your app, plus all the routes result in quite a lot of repeated code. You should instead achieve this using a parametric endpoint - please can you refactor to do this?

File: src/components/CategoryNavCards.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const CategoryNavCards = ({ category, setCategoryClicked }) => {
+  const { categoryChosen, setCategoryChosen } = useContext(CategoryContext);
+  const handleCategoryClick = () => {
+    setCategoryChosen(category.slug);
+  };
+
+  return (
+    <Link to={`/games/${category.slug}`} onClick={handleCategoryClick}>
+      <li className="category-option">{category.slug}</li>
```
Comment (itskatherine) [inline src/components/CategoryNavCards.jsx]: Nice use of mapping over a custom component so you don't need to manually hardcode each category. 🙌 
One small thing though (which isn't mandatory to change) at the moment the categories still look like their slugs (i.e. "push-your-luck") could you use code to reformat these category slugs so they read "Push your luck" for example? 🤔

File: src/components/CategoryNavCards.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Link } from "react-router-dom";
+import { CategoryContext } from "../contexts/CategoryMenu";
+import { useContext } from "react";
+
+const CategoryNavCards = ({ category, setCategoryClicked }) => {
+  const { categoryChosen, setCategoryChosen } = useContext(CategoryContext);
```
Comment (itskatherine) [inline src/components/CategoryNavCards.jsx]: Though this is a nice use of a context - if you used a parametric endpoint, you could instead rely on the useParams function to tell you the current category

File: src/components/CategoryNavCards.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  const handleCategoryClick = () => {
+    setCategoryChosen(category.slug);
+  };
+
+  return (
+    <Link to={`/games/${category.slug}`} onClick={handleCategoryClick}>
+      <li className="category-option">{category.slug}</li>
+    </Link>
+  );
```
Comment (itskatherine) [inline src/components/CategoryNavCards.jsx:8]: Also could you please make this list of category buttons responsive?

File: src/components/CategoryNavCards.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { Link } from "react-router-dom";
+import { CategoryContext } from "../contexts/CategoryMenu";
+import { useContext } from "react";
+
+const CategoryNavCards = ({ category, setCategoryClicked }) => {
+  const { categoryChosen, setCategoryChosen } = useContext(CategoryContext);
+  const handleCategoryClick = () => {
+    setCategoryChosen(category.slug);
+  };
```
Comment (itskatherine) [inline src/components/CategoryNavCards.jsx]: I'm getting the error that you get when lists don't have keys 🔑 
Could you please fix this?

**Review by itskatherine — COMMENTED** [review]
Hi Nico, sorry I've just read that you said you wanted to take back this PR but I've already written some feedback so thought there's no harm sending you through what I'd done so far!

**Review by aloush-dev — APPROVED** [review]

**Pull request merged**